src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +81,10 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /**
+    * Returns a modifiable list.
+    */
+    public List<Person> getInternalList() { return internalList; }
```

Comment:
This violates the coding standards! Hint: See the section on `Conditionals`, point 11